internal/js/modules/k6/browser/tests/webvital_test.go

```diff
@@ -94,6 +94,12 @@ func TestWebVitalMetric(t *testing.T) {
 func TestWebVitalMetricNoInteraction(t *testing.T) {
 	t.Parallel()
 
+	// Increase timeout for Windows
+	timeout := 5 * time.Second
+	if runtime.GOOS == "windows" {
+		timeout = 10 * time.Second
+	}
```

Comment:
Do we really need to differentiate? In the end, it is a timeout, so it doesn't count too much on the real execution for the average run.

It counts only on failures but they shouldn't be the average daily run.

Reply:
>I don't see a good reason not to use common.DefaultTimeout for example, which is 30s.

This is another good observation :+1: I'm for it.